rotkehlchen/accounting/rules.py

```diff
@@ -0,0 +1,170 @@
+from typing import TYPE_CHECKING, Optional
+
+from rotkehlchen.accounting.structures.base import HistoryBaseEntry
+from rotkehlchen.accounting.structures.evm_event import EvmEvent
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.evm.accounting.structures import BaseEventSettings, TxAccountingTreatment
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.db.constants import NO_ACCOUNTING_COUNTERPARTY
+
+if TYPE_CHECKING:
+    from rotkehlchen.accounting.pot import AccountingPot
+    from rotkehlchen.chain.evm.accounting.aggregator import EVMAccountingAggregators
+    from rotkehlchen.db.dbhandler import DBHandler
+
+
+def get_accounting_rule_identifier(
+        event_type: HistoryEventType,
+        event_subtype: HistoryEventSubType,
+        counterparty: Optional[str] = None,
+) -> str:
+    key = f'{event_type.serialize()}_{event_subtype.serialize()}'
+    if counterparty is not None:
+        key += f'_{counterparty}'
+
+    return key
+
+
+def make_default_accounting_settings(pot: 'AccountingPot') -> dict[str, BaseEventSettings]:
+    """
+    Returns accounting settings for events that can come from various decoders and thus don't have
+    any particular protocol. These settings also allow users to customize events in the UI.
+    TODO: Remove this hardcoded rules and add them to the database along the evm modules rules.
+    """
+    result = {}
+    result[get_accounting_rule_identifier(HistoryEventType.SPEND, HistoryEventSubType.FEE, CPT_GAS)] = BaseEventSettings(  # noqa: E501
```

Comment:
not checking validity of those. I suppose you copied them from before, right?

Reply:
yes copied from before